index.src.html

```diff
@@ -822,10 +830,10 @@ <h4 id="clear-dom">
      
       1.  An origin's WebSQL databases [[!WEBDATABASE]].
       2.  An origin's filesystems [[!file-system-api]]
-      3.  ChannelID
-      4.  Plugin data (e.g. <a href="https://wiki.mozilla.org/NPAPI:ClearSiteData">NPP_ClearSiteData</a>)
-      5.  Appcache.
-      6.  ISSUE: Moar?
+      3.  Plugin data (e.g. <a href="https://wiki.mozilla.org/NPAPI:ClearSiteData">NPP_ClearSiteData</a>),
+          except Flash [[!FLASH]] LSOs (see step #7 in [[#clear-cookies]]).
```

Comment:
Sorry, bad example. I think it's not eTLD+1, but hostname. So port and scheme can vary, but not subdomains.